src/aap_eda/api/views/credential.py

```diff
@@ -117,3 +126,25 @@ def get_serializer_class(self):
 
     def get_response_serializer_class(self):
         return serializers.CredentialSerializer
+
+    def destroy(self, request, *args, **kwargs):
```

Comment:
@Alex-Izquierdo We need to change urls.py to add this endpoint, right?

Reply:
This is not a new endpoint, just we are overriding the delete logic.